components/match2/wikis/halo/match_summary.lua

```diff
@@ -73,7 +74,8 @@ function p.getByMatchId(args)
 	for _, game in ipairs(match.games) do
 		if game.map then
 			local mapDisplay = '[[' .. game.map .. ']]'
-			local modeIcon = MapModes.get({ mode = game.mode, date = match.date, size = 15})
+			local date = Date.formatTimestamp('Y-m-d', match.date or 0)
+			local modeIcon = MapModes.get({ mode = game.mode, date = date, size = 15})
```

Comment:
might be worth a shot to check if maybe the mapModes module can be adjusted instead

Reply:
I'm sure it can be adjusted, but I feel that it should be left out of scope for this PR.